lib/browser.ts

```diff
@@ -612,8 +611,7 @@ export class Browser {
                       window['NG_PENDING_TIMEOUTS'] = {};
                     }
 
-                    let extendedTimeout: any = function() {
-                      let args = Array.prototype.slice.call(arguments);
+                    let extendedTimeout: any = (...args: any[]) => {
```

Comment:
These changes will cause issues because this code is being sent to the browser and run there. I assume this is clang format? We could move this whole module into lib/clientSideScripts.js (and probably should do that anyway).